code/game/gamemodes/changeling/generic_equip_procs.dm

```diff
@@ -32,7 +32,7 @@
 		return 1
 
 	if(M.head || M.wear_suit) //Make sure our slots aren't full
-		src << "<span class='warning'>We require nothing to be on our head, and we cannot wear any external suits.</span>"
+		src << "<span class='warning'>We require nothing to be on our head, and we cannot wear any external suits, or shoes.</span>"
```

Comment:
The shoes on the spacesuits should just be children of magboots, if I remember correctly. Means that this last line isn't true for those powers.